static/src/javascripts/projects/common/modules/ui/cmp-ui.js

```diff
@@ -69,6 +88,7 @@ const prepareUi = (): void => {
     const iframe = document.createElement('iframe');
     iframe.src = cmpConfig.CMP_URL;
     iframe.className = IFRAME_CLASS;
+    iframe.tabIndex = 1;
```

Comment:
I've included this so user's navigating with a keyboard can tab into the CMP UI.